schema/bom-1.7.proto

```diff
@@ -559,6 +559,11 @@ enum LicensingTypeEnum {
 }
 
 message Metadata {
+  message DistributionConstraints {
+    // The Traffic Light Protocol (TLP) classification that controls the sharing and distribution of the data that the BOM describes.
+    optional TlpClassification tlp = 1;
```

Comment:
`tlpClassification` or `tlpLabel` is a bit verbose but could be easy to understand without the need for a help text.